vscode-extension/src/test/extension.test.ts

```diff
@@ -0,0 +1,22 @@
+//
+// Note: This example test is leveraging the Mocha test framework.
+// Please refer to their documentation on https://mochajs.org/ for help.
+//
+
+// The module 'assert' provides assertion methods from node
+import * as assert from 'assert';
+
+// You can import and use all API from the 'vscode' module
+// as well as import your extension to test it
+// import * as vscode from 'vscode';
+// import * as myExtension from '../extension';
+
+// Defines a Mocha test suite to group tests of similar kind together
+suite("Extension Tests", function () {
```

Comment:
What do I need to do run the tests? Please add instructions

Reply:
Actually these are the default tests. They used mocha, while we plan to use jest in the future. 
We have an open issues for testing:
https://github.com/TheZoker/jsonforms-tooling/issues/20

For now I will remove the test, because they are not tests at all. When we have tests, we'll add instructions on how to run them.